tests/test_create_sentence.rs

```diff
@@ -0,0 +1,29 @@
+extern crate reqwest;
+
+use std::collections::HashMap;
+
+use reqwest::StatusCode;
+
+mod db;
+
+#[test]
+fn test_post_sentence_returns_200() {
+
+    let connection = db::get_connection();
+    db::clear(&connection);
+
+    let mut json = HashMap::new();
+    json.insert("text", "This is a sentence.");
+    json.insert("iso639_3", "en");
```

Comment:
iso639_3 is actually a 3 letter code (hence the "_3" )  

https://en.wikipedia.org/wiki/ISO_639-3

so actually this one should not work

Reply:
(but of course it is accepted because I haven't added strong constraint yet)